src/plugin/file/csv/csvdescriptor.js

```diff
@@ -124,6 +127,31 @@ plugin.file.csv.CSVDescriptor.prototype.setUseHeader = function(useHeader) {
 };
 
 
+/**
+ * @inheritDoc
+ */
+plugin.file.csv.CSVDescriptor.prototype.getExporter = function() {
+  return new plugin.file.csv.CSVExporter();
```

Comment:
I didn't consider that, but it might save a little bit of processing. Only concern would be if there's some level of statefulness to the exporter that could cause issues. `ExportManager` for example keeps instances around, but it reconstructs new instances when you call `getExportMethods()`.

Reply:
In theory you should just need to call the `reset` function on the exporter to get back to the default state. This doesn't seem like a notable amount of overhead, just a thought if it's a relatively simple change.